Lib/test/test_compileall.py

```diff
@@ -502,19 +502,19 @@ def setUp(self):
         self.directory = tempfile.mkdtemp()
         self.source_path = os.path.join(self.directory, '_test.py')
         with open(self.source_path, 'w', encoding='utf-8') as file:
-            file.write('# -*- coding: utf-8 -*-\n')
-            file.write('print u"\u20ac"\n')
+            file.write('print(b"\u20ac")\n')  # intentional error
```

Comment:
Please elaborate why it's an error, it's non obvious to me. Apparently, you expect the error:

```
SyntaxError: bytes can only contain ASCII literal characters
```

Example:

```suggestion
            # Intentional syntax error: bytes can only contain
            # ASCII literal characters.
            file.write('print(b"\u20ac")\n')
```